builders/build/substrate-api/chopsticks.md

```diff
@@ -153,23 +152,26 @@ npx @acala-network/chopsticks run-block --endpoint wss://wss.api.moonbeam.networ
 To test out XCM messages between networks, you can fork multiple parachains and a relay chain locally. For example, the following will fork Moonriver, Karura, and Kusama given that you've downloaded the [config folder from the GitHub repository](https://github.com/AcalaNetwork/chopsticks/tree/master/configs){target=_blank}:  
```

Comment:
could probably add a bit more clarity here and say that you've downloaded the entire `configs` directory or at the very least the config for Moonriver, Karura, and Kusama.